service/routes/test.js

```diff
@@ -40,30 +42,32 @@ function createEndpoint(type, polyfillio) {
 
 				// Fetch polyfill configs for all the features to be tested
 				return Promise.all(featuresList.map(featureName => {
-					return polyfillio.describePolyfill(featureName).then(config => ({[featureName]: config}) );
-				})).then(featureObjs => Object.assign({}, ...featureObjs));
+					return Promise.resolve()
+						.then(() => {
+
+						const config = polyfillio.describePolyfill(featureName);
+
+						const isTestable = config.isTestable;
+						const isPublic = config.isPublic;
```

Comment:
Maybe no need to reassign these anymore?  It now feels inconsistent with the treatment of `config.hasTests`